blog/2022/12-7-first-look-at-resoto-dashboards/index.md

```diff
@@ -0,0 +1,76 @@
+---
+authors: [pablo]
+tags: [ui, dashboard, visualization]
+---
+
+# First look at resoto dashboards
+
+Resoto makes it easier than ever to create an inventory of your cloud infrastructure. And with just a few commands, you can immediately get a snapshot of what is currently happening with your resources. Furthermore, with this data ingested into a time series database, you not only know the current state of your cloud but also how they evolve.
+
+So, with all this data now available, you will probably want a simple and human-readable way to visualize it. Until now, you could [export the data to google sheets](https://resoto.com/docs/how-to-guides/data-export/export-data-to-google-sheets) to create some charts out of it or use [resoto notebook](https://resoto.com/docs/reference/notebook) to interactively visualize and analyze the result of your searches.
+
+Now, we are introducing a new way to have this visual information always available: Resoto Dashboards.
+
+You may be wondering: "But why do we need yet another way to visualize data?". With our dashboards, you don't just visualize the data. Your searches and queries will be executed periodically and the results displayed in widgets automatically, so you always look at updated information. You can organize your widgets freely in the dashboard, to keep the most important metrics always within reach. You can share these dashboards across your organization so everybody that needs this information can make use of it. And of course, you can access your dashboards from everywhere as long as you have access to your resoto installation.
+
+## Key concepts
+
+A _dashboard_ is a container where you can place different _widgets_ to show relevant information. Each _widget_ has its way to display data, and thus, its configurable properties. Available widgets so far are:
```

Comment:
maybe `container` is a poor choice here, since the word has some pretty fixed connotations as in Docker container. Here, it's used in it's generic form, but it might be misunderstood at first. How about `.. is a place where you can put different widgets ...`?